adzerk/api.py

```diff
@@ -49,9 +49,9 @@ def __add_targeting(self, body):
             body['user'] = {'key': self.pocket_id}
         keywords = []
         if self.country is not None:
-            keywords.append('country-' + self.country)
-        if self.region is not None:
-            keywords.append('region-' + self.region)
+            keywords.append(self.country)
+            if self.region is not None:
+                keywords.append(self.country + '-' + self.region)
```

Comment:
A region without a country is ambiguous, so we shouldn't set it if the country is unavailable.

Reply:
Minor, but could simplify this check with `if self.region:` and `if self.country:` which will check for both null and empty values.